### PR TITLE
Fix cmake warnings

### PR DESCRIPTION
**Pull request by ligurio**
~~**Must be merged after #79**~~

### CODE REVIEW

File: cmake/FindTarantool.cmake [hunk 1]
```diff
@@ -23,7 +23,7 @@ if(TARANTOOL_INCLUDE_DIR)
 endif()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(TARANTOOL
+find_package_handle_standard_args(Tarantool
```
Comment (Totktonada) [inline cmake/FindTarantool.cmake:26]: It seems, here we meet the same problem as one that was fixed in https://github.com/tarantool/modulekit/pull/6

Comment (Totktonada) [reply cmake/FindTarantool.cmake:26]:
```sh
$ ag find.req
CMakeLists.txt
12:set(TARANTOOL_FIND_REQUIRED ON)
```

Comment (ligurio) [reply cmake/FindTarantool.cmake:26]: Thanks! Fixed in a separate commit.

**Review by Totktonada — APPROVED** [review]
LGTM.

**Pull request merged**